java/arcs/android/devtools/DevToolsService.kt

```diff
@@ -47,8 +47,8 @@ class DevToolsService : Service() {
     private val forwardProxyMessage = object : IStorageServiceCallback.Stub() {
         override fun onProxyMessage(proxyMessage: ByteArray) {
             scope.launch {
-                val actualMessage = proxyMessage.decodeProxyMessage()
-                devToolsServer.send(actualMessage.toString())
+                val rawMessage = RawDevToolsMessage(proxyMessage.decodeProxyMessage().toString())
```

Comment:
We'll need to properly serialize the proxy message to JSON.. simply calling `toString` will be a dump of whatever the implementation of `toString` returns, rather than something appropriate for a downstream client expecting something well-formatted.

Reply:
This is a way @sjmiles and I came up with to send well-defined messages ASAP. The notion is that we are sending the raw CRDT message, even though it isn't in a great format. The next step will be to implement a StoreMessage that passes a more refined message. By having the kind field, we can send different types of messages. The RawMessage is just a simple one to get us started.

Reply:
Ah, ok